src/v2/controllers/FormController.ts

```diff
@@ -152,6 +153,9 @@ export default Controller.extend({
   // eslint-disable-next-line max-statements
   async handleInvokeAction(actionPath = '', actionParams = {}) {
     const { appState, settings } = this.options;
+    const ignoreFormErrors = (actionParams as any)[ACTION_PARAMS_IGNORE_FORM_ERRORS];
+    // prevent passing ACTION_PARAMS_IGNORE_FORM_ERRORS in network payload
+    delete actionParams[ACTION_PARAMS_IGNORE_FORM_ERRORS];
```

Comment:
Instead of doing this explicitly just for the ignore form errors params, can we do one of two things to make this more maintainable/scalable?

1. Create a list of actionParams constants that are allowed to be passed to `invokeAction` and filter the params with the list before calling `invokeAction`
2. Nest the current `actionParams` values inside a property that indicates the params we want to send versus other params that shouldn't be sent. i.e.:
```ts
type IdxRequestParamKeys = 'reason' | 'statusCode';

type ActionParams = {
  // this type could be made even stricter by setting allowed values for each possible key (e.g. `reason` can only be one of 3 values, `statusCode` is a number
  idxRequestParams: Record<IdxRequestParamKeys,string>;
  ignoreFormErrors: boolean;
}
```

Reply:
@shawyu-okta  Updated by passing ignoreFormErrors as a separate arguments during `appState.trigger('invokeAction')` to avoid specific param handling for `actionParams`